### PR TITLE
Fixing one test case

### PR DESCRIPTION
**Pull request by zhaoyuango**
This Pull request adds one assertion to the `OzHashSpec` test suite for better readability.

### CODE REVIEW

File: src/test/java/io/vacco/oruzka/OzHashSpec.java [hunk 1]
```diff
@@ -10,14 +10,19 @@
...
 public class OzHashSpec {
   static {
     describe("Hashing", () -> {
       it("can hash an array of strings", () -> {
-        OzArrays.toStringConcat("Hello", "World")
-            .ifPresent(bytes -> System.out.println(OzMd5.md5SumOf(bytes)));
+        Object[] strings = new Object[] {"Hello", "World"};
+        OzArrays.toStringConcat(strings)
+            .ifPresent(bytes -> {
+              String md5 = OzMd5.md5SumOf(bytes);
```
Comment (jjzazuet) [inline src/test/java/io/vacco/oruzka/OzHashSpec.java]: Can we change this into a `var` style declaration? Thanks

**Review by jjzazuet — CHANGES_REQUESTED** [review]
Let's make a few changes. Thanks!

File: src/test/java/io/vacco/oruzka/OzHashSpec.java (hunk 1 above)
Comment (zhaoyuango) [reply src/test/java/io/vacco/oruzka/OzHashSpec.java]: Ok but why do we need to do that?

Comment (jjzazuet) [reply src/test/java/io/vacco/oruzka/OzHashSpec.java]: To improve code readability. Thanks

Comment (zhaoyuango) [reply src/test/java/io/vacco/oruzka/OzHashSpec.java]: I think it looks good now.

**Review by jjzazuet — APPROVED** [review]
Looks good to me

**Pull request merged**